src/MessagingAvatar/MessagingAvatar.tsx

```diff
@@ -28,27 +28,33 @@ export const MessagingAvatar: React.FC<Props> = ({ className, text, color }: Pro
       // If all else fails, we can use text as the seed to determine color.
       style={{
         backgroundColor:
-          "color" in color
-            ? color.color
-            : `hsl(${_determineAvatarHue(color.seed || text)}, 80%, 85%)`,
+          "color" in color ? color.color : `#${_determineAvatarColor(color.seed || text)}`,
```

Comment:
Wondering if this could be simplified to: `color.color || `#${_determineAvatarColor(color.seed || text)}`?

Reply:
You're correct! It wasn't possible with the typing of the prop before (TS would complain), but it works now. 🙂